ui-modules/utils/logbook/logbook.js

```diff
@@ -177,9 +177,27 @@ export function logbook() {
 
             logbookApi.logbookQuery(params, true).then((logEntries) => {
 
-                // TODO: implement logic for make output as table.
+                if ($scope.isLatest && $scope.logEntries.length !== 0) {
+                    if (logEntries.length > 0) {
+
+                        // Calculate date-time to display up to. Note, calendar does not take into account milliseconds,
+                        // round down to seconds.
+                        let latestDateTimeToDisplay = Math.floor(logEntries.slice(-1)[0].datetime / DEFAULT_NUMBER_OF_ITEMS) * DEFAULT_NUMBER_OF_ITEMS;
+
+                        // Display new log entries.
+                        let newLogEntries = logEntries.filter(entry => entry.datetime <= latestDateTimeToDisplay);
```

Comment:
Would be a bit cleaner as
`const newEntries = logEntries.filter(({datetime})=> datetime <= latestDateTimeToDisplay);`

Reply:
Noted